user/validation.py

```diff
@@ -0,0 +1,37 @@
+from django.core.exceptions import ValidationError
+import re
+
+def validate_phone(value):
+
+    regex = re.compile(r'(\d{3}).*(\d{4}).*(\d{4})')
+
+    if not regex.match(value):
+        raise ValidationError(message = 00)
+
```

Comment:
```suggestion
def validate_phone(value):
    regex = re.compile(r'(\d{3}).*(\d{4}).*(\d{4})')
    if not regex.match(value):
        raise ValidationError(message = 00)

```

공백이 너무 많아 가독성이 떨어집니다. 함수 아래 공백들은 지워주세요 아래 다른 코드도 마찬가지 입니다.